src/app/courses-list/courses-list.component.html

```diff
@@ -1,7 +1,21 @@
 <div class="courses-list">
-    <app-search></app-search>
-        <app-course *ngFor="let course of courses$ | async" [course]="course" (removeItem)="onNotify($event)"></app-course>
-    <div class="load-more">
-        <button (click)="loadMore()">Load More</button>
-    </div>
+    <app-search (notify)="onSearchNotify($event)"></app-search>
+<!--    @TODO: get rid of two subscriptions to courses$-->
+<!--    @TODO: and why ngif didn't worked with ng-template -->
+    <ng-container *ngIf="(courses$ | async ).length; else noCoursesFound">
+        <app-course
+            *ngFor="let course of courses$ | async |  filter: filterString | orderBy"
```

Comment:
so, to use the pipe in the component, it was needed to inject pipe in a component, add the pipe to component's module in the providers sections and use transform method of the pipe inside component's code